Cargo.toml

```
@@ -17,4 +17,5 @@ regex = "1"
 lazy_static = "1.4.0"
 
 [features]
-nightly = []
\ No newline at end of file
+nightly = []
+relative-path = [ "nightly" ]
```

Could we please remove the enabling of the nightly feature? This will make relative-path standalone and avoid any future side effects.

```suggestion
relative-path = []
```